crates/api/src/local_user/reset_password.rs

```diff
@@ -6,23 +6,29 @@ use lemmy_api_common::{
   SuccessResponse,
 };
 use lemmy_db_views::structs::{LocalUserView, SiteView};
-use lemmy_utils::error::{LemmyErrorExt, LemmyErrorType, LemmyResult};
+use lemmy_utils::error::LemmyResult;
+use tracing::error;
 
 #[tracing::instrument(skip(context))]
 pub async fn reset_password(
   data: Json<PasswordReset>,
   context: Data<LemmyContext>,
 ) -> LemmyResult<Json<SuccessResponse>> {
-  // Fetch that email
   let email = data.email.to_lowercase();
-  let local_user_view = LocalUserView::find_by_email(&mut context.pool(), &email)
-    .await
-    .with_lemmy_type(LemmyErrorType::IncorrectLogin)?;
+  let _ = try_reset_password(&email, &context).await;
```

Comment:
Don't set this var, and its not going to run this unless you add `?`

Reply:
`let _ =` is required to make clippy not complain about the unused value, and `?` is only for returning errors which is unwanted here